deepchem/models/torch_models/gnn.py

```diff
@@ -178,6 +180,36 @@ def forward(self, data):
         return out
 
 
+class Discriminator(nn.Module):
+    """
```

Comment:
Can you add this layer to the list of layers in `layers.rst`? We may want to rename this to be something more specific like `LocalGlobalDiscriminator`

Reply:
done. I think we should maybe have a section in `layers.rst` for graph layers, or maybe torch geometric layers.